lib/ueberauth/strategy/facebook.ex

```diff
@@ -88,7 +88,7 @@ defmodule Ueberauth.Strategy.Facebook do
       image: fetch_image(user["id"]),
       last_name: user["last_name"],
       name: user["name"],
-      nickname: user["username"],
+      nickname: nil,
```

Comment:
If it doesn't exist there's no reason to keep it around, let's remove this line.